code/modules/mob/living/simple_animal/hostile/headslug.dm

```diff
@@ -23,7 +24,7 @@
 	density = FALSE
 	ventcrawler = 2
 	a_intent = INTENT_HARM
-	speed = 0.3
+	speed = 0.2
```

Comment:
is there any reason why this is not capped to .5? We have speed difference only with 0.5 changes, 3.2 and 3.0 speed is the same

there is system to compensate that with carbon/humans but not headslugs as far as i know

Reply:
Ah hold on pushing the latest commit, this was the old speed.